android/app/src/main/java/com/mattermost/rnbeta/MattermostManagedModule.java

```diff
@@ -0,0 +1,70 @@
+package com.mattermost.rnbeta;
+
+import android.app.Application;
+import android.content.Context;
+import android.os.Bundle;
+
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.modules.core.DeviceEventManagerModule;
+
+public class MattermostManagedModule extends ReactContextBaseJavaModule {
+    public static MattermostManagedModule instance;
+
+    private boolean shouldBlurAppScreen = false;
+    private ReactApplicationContext reactContext;
+
+    private MattermostManagedModule(ReactApplicationContext reactContext) {
+        super(reactContext);
+
+        this.reactContext = reactContext;
+    }
+
+    public static MattermostManagedModule getInstance(ReactApplicationContext reactContext) {
+        if (instance == null) {
+            instance = new MattermostManagedModule(reactContext);
+        }
+
+        return instance;
+    }
+
+    @Override
+    public String getName() {
+        return "MattermostManaged";
+    }
+
+    @ReactMethod
+    public void blurAppScreen(boolean enabled) {
+        this.shouldBlurAppScreen = enabled;
+    }
+
+    public boolean isBlurAppScreenEnabled() {
+        return this.shouldBlurAppScreen;
```

Comment:
Not that it matters, but `this` is 100% optional here and on line 43, because there's no locally scoped variable with the same name as the global variable that you're trying to access